specs/dotsigstore_bundle/verification_flows.md

```diff
@@ -0,0 +1,38 @@
+# Purpose
+
+Enumerate the cosign verification use-cases with various keyed/keyless
+modes and TUF configurations. Scneraios with different Rekor/Fulcio
+instances (e.g. local, staging, corporate etc) are not explicitly
+listed, as via the correct command line parameter, the client should
+be able to figure out which keys to load in the TUF repository.
+
+| Case | Keytype | Rekor entry | Root CA | Key status | Verification passes |
+|-------------|---------|-------------|---------|------------|--------------|
+| #1   | Keyless | Yes         | Fulcio  | Valid      | :white_check_mark:  |
+| #2   | User provided | No | N/A | N/A: Verifier provides a set of accepted keys | :white_check_mark: |
+| #2.1 | User provided | Yes | N/A | Not known by verifier | :no_entry_sign: |
+| #3.1 | Keyless | Yes | Fulcio | Rekor key valid during signature generation, but new Rekor key published since then. | :white_check_mark: |
```

Comment:
IMO these subcases  (3.1–3.5) for keyless are a little irrelevant to the bundle discussion. Definitely worth documenting somewhere but it's a bit of a distraction in this context.

Reply:
Yes, see comment on listing test cases to verify against.